### PR TITLE
Mirror Log Change

### PR DESCRIPTION
**Pull request by ThatDan123**
Adds the byte size for when the byte size of packet is over limit.

### CODE REVIEW

**Comment by ThatDan123** [comment]
Wait for doobly to review before merge, though it shouldnt be a problem.

**Comment by ThatDan123** [comment]
good to merge

**Review by vlandemart — APPROVED** [review]

**Pull request merged**
